entwatch/ze_fapescape_rote_v1_3.cfg

```diff
@@ -223,9 +223,9 @@
 		"chat"              "true"
 		"hud"               "true"
 		"hammerid"          "1251507"
-		"mode"              "1"
-		"maxuses"           "0"
-		"cooldown"          "0"
+		"mode"              "4"
+		"maxuses"           "8"
+		"cooldown"          "10"
 		"maxamount"         "1"
 	}
-}
\ No newline at end of file
+}
```

Comment:
None of these items are serious, and enabling them to show usages in chat just creates useless chat spam.